csv-data-management/info.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "csv-data-management",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "label": "CSV Data Management",
   "description": "CSV Data management can perform different operations on CSV files like read file, perform deduplication, merge two CSV files,join two CSV files,concat two CSV files and return well formatted dataset",
```

Comment:
files, join two CSV files,concat two CSV files, and return formatted dataset
(spaces after commas)

Reply:
Done